shortcircuit-api/src/test/resources/FaultParametersFileInvalid.json

```diff
@@ -2,12 +2,12 @@
   "unexpected" : "f00"
 }, {
   "id" : "f01",
-  "withVoltageMap" : true
+  "withVoltageProfileResult" : true
```

Comment:
We keep that reference to `Profile` too? Why?

Reply:
This is a mistake, it should be withVoltageResult